source/usage-examples/updateMany.txt

```diff
@@ -0,0 +1,26 @@
+=====================
+Update Many Documents
```

Comment:
Changing "Many" to "Multiple" might make this a little more generic and less of a reference discussing updateMany exclusively.

Reply:
Fixed